armi/reactor/reactors.py

```diff
@@ -1160,6 +1160,15 @@ def getFirstAssembly(self, typeSpec=None, exact=False):
         """
         Gets the first assembly in the reactor.
 
+        Warning
+        -------
```

Comment:
`Warnings` is a [valid numpydoc section heading](https://numpydoc.readthedocs.io/en/latest/format.html#method-docstrings) but `Warning` is not. 

Could also use `.. warning::`

Reply:
Strange, it built just fine... Updating anyways